code/game/gamemodes/scoreboard.dm

```diff
@@ -177,26 +188,19 @@
 
 	dat += {"
 	<b><u>General Statistics</u></b><br>
-	<u>The Good:</u><br>
-
-	<b>Useful Items Shipped:</b> [GLOB.score_stuffshipped] ([GLOB.score_stuffshipped * 5] Points)<br>
-	<b>Hydroponics Harvests:</b> [GLOB.score_stuffharvested] ([GLOB.score_stuffharvested * 5] Points)<br>
-	<b>Ore Mined:</b> [GLOB.score_oremined] ([GLOB.score_oremined * 2] Points)<br>
```

Comment:
This is also tracked. In `ore_coins.dm`, `GLOB.score_oremined` is incremented when a new ore is spawned.
```js
/obj/item/stack/ore/New(loc, new_amount, merge = TRUE)
	..()
	pixel_x = rand(0, 16) - 8
	pixel_y = rand(0, 8) - 8
	if(is_mining_level(z))
		GLOB.score_oremined++ //When ore spawns, increment score.  Only include ore spawned on mining level (No Clown Planet)
```

Reply:
I know, hence why it's still in there.

Reply:
Oop yeah thats my mistake, didn't pay attention. It wasn't removed even though it was in the red section.